pom.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.mskcc.cmo</groupId>
+  <artifactId>irt-publisher</artifactId>
+  <name>CMO IRT Publisher Master</name>
+  <version>0.0.1-SNAPSHOT</version>
+  <description>IRT Sample Status Publisher project for Spring Boot</description>
+  <packaging>jar</packaging>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.3.3.RELEASE</version>
+  </parent>
+
+  <properties>
+    <java.version>1.8</java.version>
+    <maven.compiler.version>1.8</maven.compiler.version>
+    <spring.boot.version>2.3.3.RELEASE</spring.boot.version>
+    <spring.version>5.2.8.RELEASE</spring.version>
```

Comment:
yeah I think for now let's just revert to 5.2.6.RELEASE to be consistent with the other projects and then we can upgrade at a later time

Reply:
Ok, I'll downgrade.

Reply:
Done.